frontend/src/pages/Contracts/Contract/Amendments/index.tsx

```diff
@@ -22,12 +21,18 @@ export const Amendments = () => {
     deleteOne: `amendments/{id}`,
   };
```

Comment:
The way I've handled this before (where an add request needs an additional id not from the form) is to put it in the route. So the addOne url would be `contracts/${id}/amendments`. Which approach should we go with?